pip/req/req_install.py

```diff
@@ -778,6 +779,14 @@ def install(self, install_options, global_options=(), root=None):
             self.install_succeeded = True
             return
 
+        # Extend the list of global and install options passed on to
+        # the setup.py call with the ones from the requirements file.
+        # Options specified in requirements file override those
+        # specified on the command line, since the last option given
+        # to setup.py is the one that is used.
```

Comment:
@pfmoore just want to make sure you understand that overriding only works here due to the 2nd option overriding the first.

e.g.  if you specified "--install-lib" in the requirements file and the command line, both flags would be present in the "setup.py install" call, and the override would just work due to us assuming setuptools handles the options by ignoring the first.

although I can't disprove this won't work, if I were doing it, I would go to the trouble of removing the duplicate from the call

your opinion?